docs/keycodes.md

```diff
@@ -211,13 +211,15 @@ See also: [Basic Keycodes](keycodes_basic)
 |`KC_ASSISTANT`          |`KC_ASST`                      |Launch Context-Aware Assistant         |✔            |             |                 |
 |`KC_MISSION_CONTROL`    |`KC_MCTL`                      |Open Mission Control                   |             |✔            |                 |
 |`KC_LAUNCHPAD`          |`KC_LPAD`                      |Open Launchpad                         |             |✔            |                 |
+|`KC_GLOBE`              |`KC_GLB`                       |Apple Globe/Fn                         |             |✔<sup>7</sup>|                 |
 
 <sup>1. The Linux kernel HID driver recognizes [nearly all keycodes](https://github.com/torvalds/linux/blob/master/drivers/hid/hid-input.c), but the default bindings depend on the DE/WM.</sup><br/>
 <sup>2. Treated as F13-F15.</sup><br/>
 <sup>3. Must be held for about three seconds, and will display a prompt instead.</sup><br/>
 <sup>4. Holding Shift+Option allows for finer control of volume level.</sup><br/>
 <sup>5. Skips the entire track in iTunes when tapped, seeks within the current track when held.</sup><br/>
 <sup>6. WMP does not recognize the Rewind key, but both alter playback speed in VLC.</sup>
+<sup>7. The Globe key requires `GLOBE_KEY_ENABLE = yes` to be set in `rules.mk`. When pressed it will reveal the emoji picker, and it can be combined with other keys. Note however that Globe+F-keys or Globe+arrow do not work.</sup>
```

Comment:
```suggestion
<sup>7. Requires `KEYBOARD_SHARED_EP = yes` in `rules.mk` or `usb.shared_endpoint.keyboard=true` in `keyboard.json`. Globe+F-keys/arrow keys do not work.</sup>
```
Specific functionality of the Globe key depends on the OS setting.
This could be made a little more concise by adding proper documentation+explanation for the shared endpoint stuff and linking to it instead... but that's a separate issue.